cala-ledger/src/account_set/repo.rs

```diff
@@ -116,21 +116,32 @@ impl AccountSetRepo {
                 member_created_at: last.created_at.expect("created_at not set"),
             });
         }
-        let ids = rows
+        let mut account_ids = Vec::new();
+        let mut account_set_ids = Vec::new();
+
+        for row in rows.into_iter() {
+            if let Some(member_account_id) = row.member_account_id {
+                account_ids.push((
```

Comment:
why not instantiate the proper type here?

Reply:
i was  not sure if we could sort the struct... But can try that